apps/src/StudioApp.js

```diff
@@ -532,6 +503,49 @@ StudioApp.prototype.init = function(config) {
 };
 
 /**
+ * Sets html for prompts below playspace, anigif, and shows instructions dialog
+ */
+StudioApp.prototype.configureAndShowInstructions_ = function (config) {
```

Comment:
We need a type name for our `config` thing just so when I see this argument I know it's _all_ of the config options.  Maybe wsogmm (the Whole Sort Of General Mish-Mash).